com.civicrm.hrjobroles/js/src/job-roles/controllers/job-roles.controller.js

```diff
@@ -872,10 +801,7 @@ define([
           });
         })
         .then(function () {
-          return getFundersContacts();
-        })
-        .then(function (contacts) {
-          fundersContacts = contacts;
+          fundersContacts = vm.FundersData;
```

Comment:
Why do we need two storages for the same data?

Reply:
This was initially implemented in https://github.com/compucorp/civihr/pull/1279 because the pool of contacts pre-fetched by the lookup dropdown is limited to 25. We can do without this because available option values were all prefetched. References will be removed.